core/kazoo_globals/src/kz_nodes.erl

```diff
@@ -78,11 +83,16 @@
 -define(SIMPLE_ROW_STR, ?HEADER_COL ": ~s~n").
 -define(SIMPLE_ROW_NUM, ?HEADER_COL ": ~B~n").
 
+-define(NODE_GEN_SECRET, kz_binary:rand_hex(16)).
```

Comment:
This is not useful at all really, if we have to fallback to the default and its random we will have no way of knowing what the correlation is.  We should change this to something that we can determine at a later point in time if we need to.